db/migrate/20190806191648_remove_duplicate_plugin_id.rb

```diff
@@ -0,0 +1,36 @@
+class RemoveDuplicatePluginId < ActiveRecord::Migration
+
+  def up
+    # need to go through all EmbeddablePlugins and make sure each of their
+    # plugins correctly sets the plugin_scope field on the plugin
+    # we cannot easily use the polymorphic support of rails because we are also using
+    # temporary model classes, so plugin_scope_type added by the polymorphic code would
+    # end up being:  RemoveDuplicatePluginId::Embeddable::EmbeddablePlugin
+    execute <<-SQL
```

Comment:
These queries look good - the inner join with the plugin on the "left" guarantees no nulls in the embeddable_plugins.